modules/Dial_Display.js

```diff
@@ -0,0 +1,59 @@
+let dialDisplayGenerator = function(options) {
+  "ram";
+  const SCREEN_W = g.getWidth();
+  const SCREEN_H = g.getHeight();
+
+  options = Object.assign(
+    { stepsPerWholeTurn : 7, // 7 chosen as it felt the best in use.
+      dialRect : {
+        x: 0,
+        y: 0,
+        w: SCREEN_W,
+        h: SCREEN_H,
+      },
+    }, options);
+
+  const DIAL_RECT = options.dialRect;
+
+  const CENTER = {
+    x: DIAL_RECT.x + DIAL_RECT.w / 2,
+    y: DIAL_RECT.y + DIAL_RECT.h / 2,
+  };
+
+  let dialDisplay = function(step, value, isReinit) {
+    let prevValue = this.value;
+    if (value!==undefined) this.value = value;
+    if (!this.value) this.value = 0;
+    if (this.isFirstDraw===undefined || isReinit) this.isFirstDraw = true;
+    this.value += step;
+    //g.setFont("Vector:30");
+    //g.drawString(this.value);
+
+    let drawCircle = (value, R, G, B, rad, isFill)=>{
+      let x = CENTER.x+27*Math.sin(value*(2*Math.PI/options.stepsPerWholeTurn));
+      let y = CENTER.y-27*Math.cos(value*(2*Math.PI/options.stepsPerWholeTurn));
+      g.setColor(R,G,B)
+      if (!isFill) g.drawCircle(x, y, rad);
+      if (isFill) g.fillCircle(x, y, rad);
```

Comment:
Perhaps:

```suggestion
      if (!isFill) g.drawCircle(x, y, rad);
      else g.fillCircle(x, y, rad);
```

Reply:
Ah gotya - I'll let you decide on `drawCircle`'s fate just in case you do find a use